### PR TITLE
Add FlamingPy forum and slack links to website and readme 🔗 

### LINKED ISSUE #95: Add links to forum and slack to README and website
We should add links to the new forum and the Xanadu slack so people can easily find them. I would suggest to link them in the README and on the [support page](https://flamingpy.readthedocs.io/en/latest/help/support.html) of our readthedocs site.

### PR DESCRIPTION
**Pull request by soosub**
## Context for changes

Added links to the new forum and slack for FlamingPy

- **Documentation changes**:
    * Added links on README
    * Added links on readthedocs website

## Example usage and tests

- [x] Not Applicable

## Performance results justifying changes

- [x] Not Applicable

## Workflow actions and tests

- [x] Not Applicable

## Expected benefits and drawbacks

**Expected benefits:**
- Clearer communication and better community building and outreach

**Possible drawbacks:**
-  None

## Related Github issues

Closes #95 

## Checklist and integration statements

- [x] My Python and C++ codes follow this project's coding and commenting styles as indicated by existing files. Precisely, the changes conform to given `black`, `docformatter` and `pylint` configurations.
- [x] I have performed a self-review of these changes, checked my code (including for [codefactor](https://www.codefactor.io/repository/github/xanaduai/flamingpy/branches) compliance), and corrected misspellings to the best of my capacity. I have synced this branch with others as required.
- [x] I have added context for corresponding changes in documentation and [`README.md`](README.md) as needed.
- [x] I have added new workflow CI tests for corresponding changes, ensuring codecoverage is 95% or better, and these pass locally for me.
- [x] I have updated [`CHANGELOG.md`](.github/CHANGELOG.md) following the template. I recognize that the developers may revisit [`CHANGELOG.md`](.github/CHANGELOG.md) and the versioning, and create a Special Release including my changes.

### CODE REVIEW

File: doc/help/support.rst [hunk 1]
```diff
@@ -1,10 +1,7 @@
...
-If you are having issues, please let us know by posting the issue on our GitHub issue tracker.
+You can start a general discussion and connect with our community members on our `Discussions Forum <https://discuss.pennylane.ai/c/flamingpy>`__. 
+Additionally, you can join the **#flamingpy** channel on the `Xanadu Slack <https://xanadu-quantum.slack.com>`__.
 
-- **Source Code:** https://github.com/XanaduAI/flamingpy
-- **Issue Tracker:** https://github.com/XanaduAI/flamingpy/issues
-
-
-You can also start a general discussion and connect with our community members in our `Discussions Page <https://github.com/XanaduAI/flamingpy/discussions>`__. 
+If you are having issues or want to request a new feature, please let us know by creating an issue on our `GitHub issue tracker <https://github.com/XanaduAI/flamingpy/issues>`__.
```
Comment (sduquemesa) [inline doc/help/support.rst:7]:
```suggestion
If you are having issues or want to request a new feature, please let us know by creating an issue on our `GitHub issue tracker <https://github.com/XanaduAI/flamingpy/issues>`_.
```

File: doc/index.rst [hunk 2]
```diff
@@ -71,6 +71,11 @@ Features
 * Provides a host of visualization tools for ease of verifying correctness.
 
 
+Community
+---------
+
+You can start discussions and connect with our community members on our `Discussions Forum <https://discuss.pennylane.ai/c/flamingpy>`__.
```
Comment (sduquemesa) [inline doc/index.rst:77]:
```suggestion
You can start discussions and connect with our community members on our `Discussions Forum <https://discuss.pennylane.ai/c/flamingpy>`_.
```

**Review by sduquemesa — CHANGES_REQUESTED** [review]
Thanks @soosub for leading the creation of the community channels and adding them the docs! 🔥🍰 I left some small comments, please review the [PR docs build](https://flamingpy--96.org.readthedocs.build/en/96/) to see if things are rendering as you expect.

File: doc/help/support.rst [hunk 3]
```diff
@@ -1,10 +1,7 @@
 Support
 =======
 
-If you are having issues, please let us know by posting the issue on our GitHub issue tracker.
+You can start a general discussion and connect with our community members on our `Discussions Forum <https://discuss.pennylane.ai/c/flamingpy>`__. 
+Additionally, you can join the **#flamingpy** channel on the `Xanadu Slack <https://xanadu-quantum.slack.com>`__.
```
[the remark this answers is not in the document]
Comment (soosub) [reply doc/help/support.rst:5]: I fixed this after your review, I think it should be good with the current version.

File: doc/help/support.rst (hunk 1 above)
Comment (soosub) [reply doc/help/support.rst:7]: see my comment [here](https://github.com/XanaduAI/flamingpy/pull/96/files#r904210096)

File: doc/index.rst (hunk 2 above)
Comment (soosub) [reply doc/index.rst:77]: see my comment [here](https://github.com/XanaduAI/flamingpy/pull/96/files#r904210096)

File: doc/help/frequently_encountered_errors.rst [hunk 4]
```diff
@@ -1,4 +0,0 @@
-Frequently Encountered Errors
-=============================
-
-Coming soon ...
```
[the remark this answers is not in the document]
Comment (soosub) [reply doc/help/frequently_encountered_errors.rst:4]: Yes, I removed it from `index.rst` (line 130)

**Comment by soosub** [comment]
@nariman87 can I ask you for a review?

File: README.md [hunk 5]
```diff
@@ -129,12 +128,14 @@ See our contributions policy and list of contributors to FlamingPy [here](https:
 
 ## Support
```
Comment (nariman87) [inline README.md:129]: @soosub, I had to change this part to be consistent with other Xanadu project formatting. Thanks for your initial commits.

File: doc/index.rst (hunk 2 above)
Comment (nariman87) [reply doc/index.rst:77]: @sduquemesa, a single underscore should be there, we possibly have a double there by accident. Although, I would expect the extra symbols to be harmless. I'll accept the suggestion either way.

File: doc/index.rst [hunk 6]
```diff
@@ -71,6 +71,19 @@ Features
 * Provides a host of visualization tools for ease of verifying correctness.
 
 
+Support
```
Comment (nariman87) [inline doc/index.rst:74]: @soosub, I had to switch to this style for consistency reasons again.

File: doc/help/frequently_encountered_errors.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (nariman87) [reply doc/help/frequently_encountered_errors.rst:4]: Let's remove frequent errors but keep FAQ for now.

File: doc/help/support.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (nariman87) [reply doc/help/support.rst]: As I explained double underscore is redundant, it should be `_` as @sduquemesa said.

**Review by nariman87 — APPROVED** [review]

**Comment by nariman87** [comment]
@sduquemesa to review again and merge.

**Pull request merged**